### ISSUE TITLE
Custom Icons

### ISSUE THREAD

**Issue by Vivalldi** [gitterHQ/services#54]
Would it be feasible to add icons (levels) to the custom service? That way more icons can be seen other than the "logo/chat" icon and the "error" icon. Currently I am using the custom for my own personal git server.

**Comment by trevorah** [comment]
Yeah, we would have to do some long overdue changes to the api though. At the moment we require that each service has its own icon png committed into the source, but it turns out that they are pretty much just either a favicon or an error frowny face. We could just start passing urls around...

**Comment by Vivalldi** [comment]
Alright, I guess what I was wondering is if I created a pull request with new icons for any of the services (png's committed to source) would I then be able to use them (as well as others).

**Comment by trevorah** [comment]
Not at the moment, no. Each service has its own set of icons.
